components/form/RadioButton.vue

```diff
@@ -79,7 +79,7 @@ export default {
 
 <template>
   <label
-    class="radio-container"
+    :class="[ isDisabled ? 'disabled' : '', 'radio-container']"
```

Comment:
I think this is fine but I figured I'd make a note just in case you didn't know. You can also do something like: :class="{'disabled': isDisabled, 'radio-container': true}.

Reply:
That's much cleaner, I'll tweak it